packages/www/components/Dashboard/AssetDetails/AssetPlayerBox/Progress.tsx

```diff
@@ -0,0 +1,38 @@
+import { Flex, Text } from "@livepeer/design-system";
+import Spinner from "components/Dashboard/Spinner";
+
+const Progress = ({
+  text,
+  percentage,
+}: {
+  text: string;
+  percentage?: number;
+}) => (
+  <Flex
+    direction="column"
+    gap={1}
+    align="center"
+    justify="center"
+    css={{
+      width: "100%",
+      height: 265,
+      borderRadius: "$2",
+      overflow: "hidden",
+      bc: "#28282c",
+    }}>
+    <Spinner
+      css={{
+        color: "$whiteA10",
+        width: "$9",
+        height: "$9",
+      }}
+    />
+    {percentage !== undefined && (
+      <Text size="2" css={{ color: "$whiteA12" }}>
+        {text} {percentage}%
```

Comment:
WDYT of omitting the `%` when it is `0`? 
```suggestion
        {text}{percentage ? ` ${percentage}%` : '...'}
```

Reply:
Greeat suggestion. Small thing but it makes sense!